src/app/modules/shared-components/shared-components.module.ts

```diff
@@ -87,13 +88,14 @@ import { ErrorComponent } from './components/error/error.component';
     SubSectionComponent,
     AddContentComponent,
     FloatingSaveComponent,
-    ProgressSectionComponent,
-    BooleanSectionComponent,
+    ProgressFormComponent,
+    BooleanFormComponent,
```

Comment:
The `Form` suffix is misleading, those are form controls and not forms, I meaning they are not a group of controls which you can submit

Reply:
Fixed, I renamed both of them:
* `progress-form` is now `progress-select`
* `boolean-section` is now `switch-select`

Reply:
progress-select is OK :heavy_check_mark: 
switch-select is still weird: it's not a select, it's a switch :thinking:

Reply:
how about `switch-field`?

Reply:
OK let's try that :+1:

Reply:
fixed